pkgs/development/python-modules/mxnet/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv
+, lib
+, buildPythonPackage
+, requests
+, numpy
+, graphviz
+, onnx
+, mxnet
+, python
+, isPy3k
+}:
+
+buildPythonPackage rec {
+  inherit (mxnet) name version src meta;
+
+  propagatedBuildInputs = [ mxnet requests numpy graphviz onnx ];
+
+  LD_LIBRARY_PATH = lib.makeLibraryPath [ mxnet ];
+
+  doCheck = !isPy3k;
```

Comment:
Can you add a comment with the reason?

Reply:
Don't know the exact reason, it is from the original expression. I opportunistically remove this block now. Will be able to check it carefully in a few days.